desktop/src/com/unciv/app/desktop/DesktopLauncher.kt

```diff
@@ -36,6 +38,22 @@ internal object DesktopLauncher {
     @JvmStatic
     fun main(arg: Array<String>) {
 
+        // Find the index of the "-creategame" parameter
+        val createGameArg = arg.find { it.startsWith("--creategame=") }
+        if (createGameArg != null) {
+            val settingsPath = createGameArg.substringAfter("=")
+            val game = UncivGame(true)
+
+            UncivGame.Current = game
+            UncivGame.Current.settings = GameSettings()
```

Comment:
Oooohhhh - a review _request_ - I feel `listOf("honored", "stressed").random()`. Let's see...

Dumb idea here: Sound? Would someone wish to run this silently? Flag in your config?
A line below: Would someone wish to save time by not loading mods? Probably not.